### PR TITLE
- PEV185

### PR DESCRIPTION
**Pull request by Vladyslav225**
Removes the loaded HTML pages and the script that collected channel names from them
Сreated a parser to collect publisher id from the Google Play Store and changed the functionality of the script:
Сreated main function for:
    definition of the Google Paly with app details link and YouTube (mobile and desktop version) with video page link
    is checked whether the script is on the Google Play pages or on the YouTube 
    (mobile version or desktop version) pages and the required function is called for 
    searching data

### CODE REVIEW

File: web_extention/script.js [hunk 1]
```diff
@@ -15,14 +16,28 @@ function displayResult() {
 };
 
 function getYouTubeChanelName(callback) {
-    const getParentLinkElement = document.querySelector('yt-formatted-string.style-scope.ytd-channel-name.complex-string');
-    if(getParentLinkElement) {
-        const getChannelName = getParentLinkElement.querySelector('a.yt-simple-endpoint.style-scope.yt-formatted-string').getAttribute('href').replace('/', '');
-        callback(getChannelName);
+    if(window.innerWidth <= 768){
```
Comment (lebeichuk) [inline web_extention/script.js:19]: we should not check screen width
check if current url is mobile version is enough 

+ it's a good idea to separate logic for mobile and desktop to separate function

File: web_extention/script.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 function displayResult() {
-    const pageYouTubePageVideo = 'https://www.youtube.com/watch?v=';
+    const desktopYouTubeVideoPage = 'https://www.youtube.com/watch?v=';
+    const mobileYouTubeeVideoPage = 'https://m.youtube.com/watch?v='
     const pageGooglePlayApp = 'https://play.google.com/store/apps/details?id=';
 
-    if(window.location.href.includes(pageYouTubePageVideo)) {
+    if(window.location.href.includes(desktopYouTubeVideoPage) || window.location.href.includes(mobileYouTubeeVideoPage)) {
```
Comment (lebeichuk) [inline web_extention/script.js:6]: the way it's all written - this check doesn't make sense
see comment below

File: web_extention/script.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+function displayResult() {
+    const desktopYouTubeVideoPage = 'https://www.youtube.com/watch?v=';
+    const mobileYouTubeeVideoPage = 'https://m.youtube.com/watch?v='
+    const pageGooglePlayApp = 'https://play.google.com/store/apps/details?id=';
```
Comment (lebeichuk) [inline web_extention/script.js:4]: here is only desktop version of page, what about mobile?

**Review by lebeichuk — CHANGES_REQUESTED** [review]
please read comments

File: web_extention/script.js (hunk 3 above)
Comment (Vladyslav225) [reply web_extention/script.js:4]: Create a Google Play parser for the mobile version?

Comment (lebeichuk) [reply web_extention/script.js:4]: it was written in a ticket that script should work in mobile as well
here script will be executed only if it's desktop version

Comment (Vladyslav225) [reply web_extention/script.js:4]: The script for the desktop version also works for the mobile version.
html for the desktop version is the same as for the mobile version.

Comment (lebeichuk) [reply web_extention/script.js:4]: and urls is the same for mobile and desktop ?

Comment (lebeichuk) [reply web_extention/script.js:4]: I see, than all good

File: web_extention/script.js (hunk 2 above)
Comment (lebeichuk) [reply web_extention/script.js:6]: ти так спитав - що відповідь буде просто за тебе написати це все

**Review by lebeichuk — APPROVED** [review]

**Pull request merged**